src/containers/datasets/national-dashboard/types.d.ts

```diff
@@ -0,0 +1,41 @@
+type Categories = Readonly<{
```

Comment:
I think you can go here with `Readonly<Record<Category, number>>`and reuse the `Category` type.